server/i18n/test/.gitignore

```diff
@@ -1,2 +0,0 @@
-output.php
```

Comment:
All of us will probably have a file at `server/i18n/test/output.php` that will now appear to be untracked with no explanation. Do you know of a solution to this?